VerifyAttachmentsActivity/index.ts

```diff
@@ -0,0 +1,29 @@
+﻿/*
+ * This function is not intended to be invoked directly. Instead it will be
+ * triggered by an orchestrator function.
+ *
+ * Before running this sample, please:
+ * - create a Durable orchestration function
+ * - create a Durable HTTP starter function
+ * - run 'npm install durable-functions' from the wwwroot folder of your
+ *   function app in Kudu
+ */
+
+import { AzureFunction, Context } from "@azure/functions";
+import * as U from "../commons/verify-utils/utils";
+
+const verifyAttachments = () =>
+  U.verifyAllAttachments().map(taskEmails => taskEmails.run());
+
+async function Main(): Promise<void> {
+  await verifyAttachments().run();
+}
+
+const verifyAttachmentsActivity: AzureFunction = async (
+  context: Context
+): Promise<void> => {
+  context.log("start activity");
+  return await Main();
```

Comment:
this method here should take the ImapOptions (do not pass them implicitely by importing a module)